Dockerfile

```diff
@@ -1,15 +1,15 @@
-FROM usgswma/python:3.8
+FROM public.ecr.aws/ubuntu/postgres:12.4-20.04_beta
```

Comment:
This seems fine, especially considering ecr is really new - I wonder if there's a non-beta version of the image we could use in the future?